reporter/html.js

```diff
@@ -1,6 +1,14 @@
 import QUnit from "../src/core";
 import { window, navigator } from "../src/globals";
 
+const stats = {
+	totalTests: 0,
```

Comment:
I'll remove `totalTests` from being tracked altogether. We only use it after the run has finished to display the number of tests that ran, so I just created a local variable at that point which clearly shows that it is the sum of the other categories.